web/impact/impact/utils.py

```diff
@@ -31,6 +32,9 @@
     "other": "o",
     "prefer not to state": "p",
 }
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
I was mimicked what the browsable API displays for datetimes.  Since this now only gets used in this one location, I've removed the constant and added a comment.